package.json

```diff
@@ -172,8 +172,7 @@
     "react-native-flipper": "^0.146.1",
     "react-native-performance-flipper-reporter": "^2.0.0",
     "react-native-svg-transformer": "^0.14.3",
-    "react-test-renderer": "16.13.1",
-    "rn-async-storage-flipper": "0.0.10",
```

Comment:
Need to look into this change

Reply:
Fixed in [e2a3b9e](https://github.com/Expensify/App/pull/9867/commits/e2a3b9e74e07088970845f5cd792ac07f67781d2)